lib/u3d/unity_versions.rb

```diff
@@ -135,7 +135,7 @@ def save_package_size(version, url)
             size = Integer(response['Content-Length'])
           end
           if size
-            INIparser.create_linux_ini(version, size)
+            INIparser.create_linux_ini(version, size, url)
```

Comment:
OK. I was trying to group the call to INIparser into the `UnityVersionDefinition` class. If unity changes its format from INI to JSON, this would change as well. So it's very similar to what the file is about.